### ISSUE TITLE
Validation and Testing

### ISSUE THREAD

**Issue by CodeLawd** [playproject-io/contracting#43]
## `@todo`
Perfrom some test and validation (5 days)
 - [ ] Ensure that the website is adaptable to all screen layout both mobile and tablet 
 - [ ] Ensure that the website is SEO friendly 
 - [ ] Usability Testing by potential end users
   - [ ]  Create Useabilty task questions
      - `@input` :question: `Link to a working website design (dat ecosystem website)`  from #42 
      - `@output` :question: `Link to usability test report`
 - Next #48
## `@info`
Testing and validating the website by users to ensure it usability and efficiency

**Comment by serapath** [comment]
**feedback `2022.04.28`**

same as in https://github.com/playproject-io/contracting/issues/45#issuecomment-1112698703

**BUT** i can see yu use `from` a bunch of times, but in issue #42 there is no output that is named: `ENsure that the website is adaptable to all screen layout both mobile and tablet` ...which anyway sounds like a todo and not the actual output document that was maybe meant to be created.

same for `send feedback`
same for `Ensure that the website is SEO friendly`

So basically these are meant to be tasks based on what you write, but you need to also define the input and output document names

**Comment by serapath** [comment]
merged into #49
